examples/vmck.nomad

```diff
@@ -0,0 +1,148 @@
+# TODO: remove hardcoded values
```

Comment:
I agree with the TODO up to now. But now, before mergeing, please create an issue in this repo, with a link to this PR, and remove the TODO. We prefer to track all issues as issues :)